test/test_simple_commands.py

```diff
@@ -328,25 +351,25 @@ def test_save_screenshot_valid(self, tmpdir):
         manager.close()
 
         # Check that viewport image is not blank
-        pattern = os.path.join(str(tmpdir), 'screenshots', '1-*-test.png')
+        pattern = os.path.join(str(self.tmpdir), 'screenshots', '1-*-test.png')
```

Comment:
Note that I'm just making use of an existing feature of `OpenWPMTest` class that sets the tmpdir as a class property.